### PR TITLE
Update bot rules descriptions and docs

### PR DESCRIPTION
**Pull request by mdanish-kh**
cc @Trenly @denelon

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/109336&drop=dogfoodAlpha

### CODE REVIEW

**Review by Trenly — APPROVED** [review]

**Review by denelon — CHANGES_REQUESTED** [review]

**Comment by denelon** [comment]
@Trenly something strange happened here. I think we want the "Needs-Author-Feedback" to persist in this scenario.

**Comment by mdanish-kh** [comment]
@denelon That's because I replied to your review inside the thread

> @Trenly something strange happened here. I think we want the "Needs-Author-Feedback" to persist in this scenario.

@denelon That's because I replied to your review inside the thread. I believe this is the expected behavior

File: .github/policies/labelAdded.validationSmartScreen.yml [hunk 1]
```diff
@@ -35,7 +35,7 @@ configuration:
               users:
                 - KevinLaMS
           - addLabel:
-              label: Needs-Attention
+              label: Needs-Attention # This will automatically assign the ICM Users
```
[the remark this answers is not in the document]
Comment (denelon) [reply .github/policies/labelAdded.validationSmartScreen.yml:38]: Yeah, we should remove the Kevin part. He was originally triaging all these, but now we shouldn't assign to Kevin. 😊

**Comment by mdanish-kh** [comment]
Removed assignment to KevinLaMS in the latest commit

**Review by denelon — APPROVED** [review]

**Pull request merged**